cmake/onnxruntime_unittests.cmake

```diff
@@ -61,9 +61,15 @@ function(AddTest)
 endfunction(AddTest)
 
 #Check whether C++17 header file <filesystem> is present
-include(CheckIncludeFiles)
-check_include_files("filesystem" HAS_FILESYSTEM_H LANGUAGE CXX)
-check_include_files("experimental/filesystem" HAS_EXPERIMENTAL_FILESYSTEM_H LANGUAGE CXX)
+if (${CMAKE_VERSION} VERSION_LESS 3.11)
```

Comment:
I guess the follow code works in all versions:

```
include(CheckIncludeFileCXX)
 check_include_files("experimental/filesystem" HAS_EXPERIMENTAL_FILESYSTEM_H LANGUAGE CXX)	   
 check_include_file_cxx("filesystem" HAS_FILESYSTEM_H)
 check_include_file_cxx("experimental/filesystem" HAS_EXPERIMENTAL_FILESYSTEM_H)
```

Therefore, we can remove this if...else... to make it simpler. What do you think?

Reply:
It will not work for older versions of CMake - there's no `CheckIncludeFiles` module for CMake version older than 3.11. But this does not matter since there's `cmake_minimum_required` in the master CMakeLists.txt set to 3.11. With this limitation, one cannot use older CMake. It has to be CMake 3.11 or newer.

Reply:
I think we can just close this PR.